### ISSUE TITLE
no selection/copy/paste in address bar

### ISSUE THREAD

**Issue by lovinrain** [mucommander/mucommander#831]
Hello, I am using M1 Mac, I found there is no option to copy and paste in the address bar (the one in "change current location). 

The shortcuts do not work and right click option won't show either.

Is this by design?

**Comment by lovinrain** [comment]
I found it was due to the `look/feel` I have chosen. I chose CDE/Motif and there is no such option.

Is this by design then?

**Comment by ahadas** [comment]
yeah, indeed it happens with CDE/Motif and doesn't happen with other L&Fs I checked - no, it's not by design
